crates/blockifier/src/execution/syscalls/hint_processor.rs

```diff
@@ -229,6 +230,9 @@ pub struct SyscallHintProcessor<'a> {
     pub read_values: Vec<Felt>,
     pub accessed_keys: HashSet<StorageKey>,
 
+    pub read_class_hash_values: Vec<ClassHash>,
```

Comment:
Is there a reason why the accessed key is a HashSet while the read values is a vector?